src/test/groovy/org/assertj/generator/gradle/SimpleBuild.groovy

```diff
@@ -159,7 +175,10 @@ class SimpleBuild {
 
         assert result.task(':generateAssertJ').outcome == TaskOutcome.SUCCESS
         assert result.task(':compileTestJava').outcome == TaskOutcome.SUCCESS
-    }
-
 
+        def packagePath = testProjectDir.root.toPath()
+                .resolve("build/generated-src/main-test/java")
+                .resolve("org/example")
+        assert !packagePath.toFile().listFiles().find { it.name == "OtherWorldAssert.java" }
```

Comment:
This assert was missing -- it was working before we ignored it, but it wasn't actually checking.